test/unit/lib/choicesOverview.js

```diff
@@ -0,0 +1,14 @@
+const choicesOverview = require('../../../app/lib/choicesOverview');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('choicesOverview', () => {
+  it('should have valid ID appendend to the URL', () => {
+    const validId = [{
+      identifier: 'FA008'
+    }];
+    const fakeUrl = choicesOverview.addUrl(validId);
```

Comment:
The returned value isn't a fakeUrl, it's an array of cut down orgs. The variable name should reflect that, it is confusing otherwise.

Reply:
Done